cmd/mockery.go

```diff
@@ -66,7 +66,7 @@ func NewRootCmd() *cobra.Command {
 	pFlags.Bool("testonly", false, "generate a mock in a _test.go file")
 	pFlags.String("case", "", "name the mocked file using casing convention [camel, snake, underscore]")
 	pFlags.String("note", "", "comment to insert into prologue of each generated file")
-	pFlags.String("cpuprofile", "", "write cpu profile to file")
+	pFlags.String("profile", "", "write cpu profile to file")
```

Comment:
We can't change the param name as that's a breaking change, even if it's not linked to anything. I looked back at v1 and it seems this was originally tied to the `--cpuprofile` flag: https://github.com/vektra/mockery/blob/v1.1.2/cmd/mockery/mockery.go#L152

I must have broken this during the v2 revamp and no one noticed until now :D

Probably what we should do is leave the `--cpuprofile` param, but check for `if r.Config.Profile || r.Config.Cpuprofile` because `profile` could still have been set in config or the environment.

Reply:
All right, I added a commit to add back `--cpuprofile`.

Reply:
Best to leave `--profile` off of the parameter list entirely, actually. It can still be set via config.

Reply:
Done.